src/DIDContract/index.ts

```diff
@@ -19,15 +14,14 @@ export default class DIDContract {
   public contract: Contract;
 
   public constructor(
-    network: string,
+    address: string,
```

Comment:
Not particularly a fan of this approach because it requires the developer to search for the address. I understand that this isn't great for local testing or for custom contracts, so I think a good approach would be to accept an `address` and a `network` parameter. If the `address` is present, it uses that. Otherwise, it uses the `network` parameter to get from the map. If none is present, throws an error.

Reply:
Hum, I agree that we can have a more configurable approach here. 
I've fixed this on my last commit.